lib/open_food_network/cached_products_renderer.rb

```diff
@@ -27,6 +27,8 @@ def products_json
     private
 
     def cached_products_json
+      return uncached_products_json unless Spree::Config[:enable_products_cache?]
+
       if Rails.env.production? || Rails.env.staging?
```

Comment:
What if we rather extract a method that takes these 3 conditions into account? I think this would decouple this method from these low-level details.

Reply:
I am merging all changes from master to 2-0 so when I work in master I avoid all refactoring 🙄 🤣 

this is very obviously a candidate for improvement, I'll do it now.

Reply:
done